w3-hw1-ThuyDung/cypress/pages/formPage.js

```diff
@@ -0,0 +1,245 @@
+export const formPage ={
+
+    TXT_FIRSTNAME: "#firstName",
+    TXT_LASTNAME: "#lastName",
+    TXT_EMAIL: "#userEmail",
+    RDO_MALE: "#gender-radio-1",
+    RDO_FEMALE: "#gender-radio-2",
+    RDO_OTHER: "#gender-radio-3",
+    TXT_MOBILE: "#userNumber",
+    TXT_DOB :"#dateOfBirthInput",
+    DDL_MONTH: ".react-datepicker__month-select",
+    DDL_YEAR: ".react-datepicker__year-select",
+    DTP_DAY: ".react-datepicker__week .react-datepicker__day",
+    TXT_SUBJECTS: "#subjectsInput",
+    CBO_SUBJECTS: ".subjects-auto-complete__menu",
+    CHK_SPORTS: "#hobbies-checkbox-1",
```

Comment:
What happen if I have 100 checkboxes? Try to make your code flexible. I see that you use once above, `cy.get().contains().click()`. It can apply for gender, city state or something else with many options to choose